### PR TITLE
Main bg

### PR DESCRIPTION
**Pull request by KonovalovaVV**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    margin: 0 auto;
+    width: 100%;
+}
+
+.section-bg {
+    display: flex;
+    flex-direction: column;
+    background-image: url(../img/bg-mountain.png);
+    background-repeat: no-repeat;
+    max-width: 1920px;
```
Comment (MaksHladki) [inline css/main.css:21]: use width: 100% instead of max-width.

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    height: 520px;
+    margin: 0 auto;
```
Comment (MaksHladki) [inline css/main.css:23]: You don't need margin: 0 auto here

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.section-bg__text {
+    margin-top: 125px;
```
Comment (MaksHladki) [inline css/main.css:27]: You don't need margin-top here.

File: css/main.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    line-height: 20px;
+    text-transform: uppercase;
+    letter-spacing: 3.2px;
+}
+
+.section-bg__icon {
+    display: flex;
+    margin: auto;
+    margin-top: 41px;
+    width: 69px;
```
Comment (MaksHladki) [inline css/main.css:58]: Please, set 70px and reuse some styles from the footer.

File: index.html [hunk 5]
```diff
@@ -41,7 +42,29 @@
...
+          Lorem ipsum laro<br />
+          dolore sit amet vivamus
+        </h1>
+        <h2 class="subtitle">
+          Lorem ipsum laro sit amet<br />
+          dolore sit ame
+        </h2>
+      </div>
+
+      <button class="section-bg__icon">
```
Comment (MaksHladki) [inline index.html:59]: add aria-label or aria-hidden attribute

File: index.html [hunk 6]
```diff
@@ -41,7 +42,29 @@
     </nav>
   </header>
 
-  <main class="content-temp"></main>
+  <main class="main">
+
+    <div class="section-bg">
+      <div class="section-bg__text">
+        <h1 class="title">
```
Comment (MaksHladki) [inline index.html:49]: Let's discuss

File: index.html [hunk 7]
```diff
@@ -41,7 +42,29 @@
...
+          Lorem ipsum laro<br />
+          dolore sit amet vivamus
+        </h1>
+        <h2 class="subtitle">
```
Comment (MaksHladki) [inline index.html:53]: I think it should be a common text, please use `p` instead.

File: css/main.css (hunk 1 above)
Comment (KonovalovaVV) [reply css/main.css:21]: Ok.

File: css/main.css (hunk 2 above)
Comment (KonovalovaVV) [reply css/main.css:23]: Ok.

File: css/main.css (hunk 3 above)
Comment (KonovalovaVV) [reply css/main.css:27]: Ok, remove.

File: css/main.css (hunk 4 above)
Comment (KonovalovaVV) [reply css/main.css:58]: Ok, add "circle-button" class.

File: index.html (hunk 7 above)
Comment (KonovalovaVV) [reply index.html:53]: Ok.

**Pull request merged**